CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [develop]
 
+### Added
```

Comment:
I would remove the added section. Even if we fixed it by adding something there is no value for this entry.

Reply:
Yes this is irrelevant to library users and it's not a new feature. The "fixed" entry is sufficient.